core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,19 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
+      skillDifficulty.setDescription('new test skill');
+      expect(skillDifficulty.getDescription()).toEqual('new test skill');
+    });
+
+    it('should set the difficulty of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
+      skillDifficulty.setDifficulty(0.5);
```

Comment:
Follow all the above comments for this test block as well! :)

Reply:
Done.